doc/README_osx.md

```diff
@@ -1,4 +1,5 @@
-Deterministic OS X Dmg Notes.
+Cross Plattform and Deterministic OS X Dmg Notes.
+================================================
 
 Working OS X DMGs are created in Linux by combining a recent clang,
 the Apple binutils (ld, ar, etc) and DMG authoring tools.
```

Comment:
Maybe add some like "or by using homebrew and the [git-jiro](https://github.com/Git-Jiro/homebrew-jiro) recipe".